support/cas-server-support-otp-mfa-core/src/main/java/org/apereo/cas/otp/util/QRUtils.java

```diff
@@ -66,7 +69,13 @@ public static void generateQRCode(final OutputStream stream, final String key,
                 .filter(j -> byteMatrix.get(i, j))
                 .forEach(j -> graphics.fillRect(i, j, 1, 1)));
 
-        ImageIO.write(image, "png", stream);
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        ImageIO.write(image, "PNG", out);
+        byte[] bytes = out.toByteArray();
+
+        String base64bytes = EncodingUtils.encodeBase64(bytes);
+        String src = "data:image/png;base64," + base64bytes;
```

Comment:
Remove `"data:image/png;base64,"` and move it over to the actual template to be consistent with other areas in CAS doing the same.

Reply:
I've pushed a fixup to address this one